### ISSUE TITLE
Issues with village pieces

### ISSUE THREAD

**Issue by InfinityRaider** [InfinityRaider/AgriCraft#62]
It causes crashes in Mystcraft ages and warnings on server starting (#61)

**Comment by InfinityRaider** [comment]
@bogatyr-n can you check if this still persists in the latest snapshot?
https://github.com/InfinityRaider/AgriCraft/releases

**Comment by bogatyr-n** [comment]
@InfinityRaider Just tried it, no luck.
Digging through the logs reveals this though:

```
[19:31:20] [Server thread/INFO] [FML/]: Loading dimension -1 (world) (net.minecraft.server.dedicated.DedicatedServer@659a6310)
[19:31:21] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:748]: java.lang.InstantiationException: com.InfinityRaider.AgriCraft.world.StructureGreenhouseIrrigated
[19:31:21] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at java.lang.Class.newInstance(Class.java:418)
[19:31:21] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at net.minecraft.world.gen.structure.MapGenStructureIO.func_143032_b(SourceFile:81)
[19:31:21] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at net.minecraft.world.gen.structure.StructureStart.func_143020_a(StructureStart.java:104)
[19:31:21] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at net.minecraft.world.gen.structure.MapGenStructureIO.func_143035_a(SourceFile:69)
[19:31:21] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at net.minecraft.world.gen.structure.MapGenStructure.func_143027_a(MapGenStructure.java:267)
[19:31:21] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at net.minecraft.world.gen.structure.MapGenStructure.func_151538_a(MapGenStructure.java:30)
[19:31:21] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at net.minecraft.world.gen.MapGenBase.func_151539_a(SourceFile:30)
[19:31:21] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at net.minecraft.world.gen.ChunkProviderGenerate.func_82695_e(ChunkProviderGenerate.java:496)
[19:31:21] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at net.minecraftforge.common.chunkio.ChunkIOProvider.callStage2(ChunkIOProvider.java:49)
[19:31:21] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at net.minecraftforge.common.chunkio.ChunkIOProvider.callStage2(ChunkIOProvider.java:12)
[19:31:21] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at net.minecraftforge.common.util.AsynchronousExecutor.skipQueue(AsynchronousExecutor.java:344)
[19:31:21] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at net.minecraftforge.common.util.AsynchronousExecutor.getSkipQueue(AsynchronousExecutor.java:302)
[19:31:21] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at net.minecraftforge.common.chunkio.ChunkIOExecutor.syncChunkLoad(ChunkIOExecutor.java:12)
[19:31:21] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at net.minecraft.world.gen.ChunkProviderServer.loadChunk(ChunkProviderServer.java:126)
[19:31:21] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at net.minecraft.world.gen.ChunkProviderServer.func_73158_c(ChunkProviderServer.java:101)
[19:31:21] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at net.minecraft.server.MinecraftServer.func_71222_d(MinecraftServer.java:265)
[19:31:21] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at net.minecraft.server.MinecraftServer.func_71247_a(MinecraftServer.java:235)
[19:31:21] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at net.minecraft.server.dedicated.DedicatedServer.func_71197_b(DedicatedServer.java:258)
[19:31:21] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:387)
[19:31:21] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685)
[19:31:21] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:748]: Caused by: java.lang.NoSuchMethodException: com.InfinityRaider.AgriCraft.world.StructureGreenhouseIrrigated.<init>()
[19:31:21] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at java.lang.Class.getConstructor0(Class.java:2971)
[19:31:21] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at java.lang.Class.newInstance(Class.java:403)
[19:31:21] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	... 19 more
```

**Comment by InfinityRaider** [comment]
N1mo fixed this in 0a6e2124c1da5f442fbb18e049f7eb89e0456fd5
